<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personalInfo</class>
 <widget class="QWidget" name="personalInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="uNameLab">
        <property name="styleSheet">
         <string notr="true">font: 700 italic 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="text">
         <string>userName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="idLabel">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="text">
         <string>ID号：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="headBtn">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Noto Sans&quot;;</string>
      </property>
      <property name="text">
       <string>地区：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="positionLab_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Noto Sans&quot;;</string>
      </property>
      <property name="text">
       <string>未知</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="transBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/转发.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="msgBtn">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/消息.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>363</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
